src/i18n/locales/en/translation.json

```diff
@@ -634,6 +634,19 @@
       },
       "success": "Withdraw request of {{amount}} sent successfully to {{sender}}"
     },
+    "lnurlredeem": {
```

Comment:
I think the key should follow our conventions, should it be an _ between words? and maybe lnurl_withdraw rather than redeem. "Redeem" is the English translation shown to users.

Reply:
we already have a translations for lnurl_withdraw for lnurlWithdraw screen. is it fine to have redeem here?
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/b34b5688-9673-4d5a-b7dd-16fcfd435674)
as other translations for lnurl don't contain underscore for the sake of consistency

Reply:
sounds ok to me, thanks